swatch-producer-aws/deploy/clowdapp.yaml

```diff
@@ -15,12 +15,34 @@ parameters:
     value: 350m
   - name: CPU_LIMIT
     value: 1500m
-  - name: ENV_NAME # naming convention is env-$BONFIRE_APP_NAME (from bonfire config yaml file)
+  - name: ENV_NAME
     value: env-swatch-producer-aws
   - name: IMAGE
     value: quay.io/cloudservices/swatch-producer-aws
-  - name: IMAGE_TAG # bonfire takes precedence and pulls latest unless the bonfire -i flag is used
+  - name: IMAGE_TAG
     value: latest
+  - name: ENABLE_SPLUNK_HEC
+    value: 'true'
+  - name: SPLUNK_HEC_URL
+    value: https://splunk-hec.redhat.com:8088
+  - name: SPLUNK_SOURCE
+    value: 'swatch-producer-aws'
+  - name: SPLUNK_SOURCE_TYPE
+    value: 'quarkus_service'
+  - name: SPLUNK_HEC_BATCH_SIZE
+    value: '10'
+  - name: SPLUNK_HEC_BATCH_INTERVAL
+    value: '10S'
+  - name: SPLUNK_HEC_RETRY_COUNT
+    value: '3'
+  - name: SPLUNK_HEC_INCLUDE_EX
+    value: 'true'
+  - name: SWATCH_INTERNAL_SUBSCRIPTION_ENDPOINT
```

Comment:
there are subtasks on the "deploy to stage with clowder" to change these last few params